roles/ansible/tower/config-ansible-tower-ocp-ldap-ca/tasks/ocp_set_ca_certificate.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+- name: Check for CA ConfigMap
+  command: |
+    oc get configmap ldap-pem \
+      -n {{ openshift_project }}
+  register: ldap_ca_check
+  failed_when: ldap_ca_check.rc > 1
+
+- name: Create and Mount CA cert as configmap
+  block:
+    - name: Create configmap for CA
+      command: |
+        oc create configmap ldap-pem \
+          --from-file={{ ansible_tower.ldap.ca_cert }} \
+          -n {{ openshift_project }}
+    - name: Mount configmap as volume in /etc/certs direcotry
+      command: |
+        oc set volume deployment/ansible-tower \
+          --add \
+          --configmap-name ldap-pem \
+          --type configmap \
+          --mount-path /etc/certs/{{ ansible_tower.ldap.ca_cert | basename }} \
+          --sub-path {{ ansible_tower.ldap.ca_cert | basename }} \
+          -n {{ openshift_project }}
+  when:
+    - ldap_ca_check.rc != 0
+
+- name: Set data for ldap.py configuration file
+  command: |
+    oc set data secret/ansible-tower-secrets \
+      --from-file={{ ansible_tower.ldap.ldap_py }} \
+      -n {{ openshift_project }}
+
+- name: Set fact for secret volume paths in /etc/tower/conf.d directory
+  set_fact:
```

Comment:
there doesn't really seem to be any "dynamic" data in this 'fact', so probably better to just add this to `defaults/main.yml` instead.